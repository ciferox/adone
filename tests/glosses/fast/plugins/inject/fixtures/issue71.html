<!DOCTYPE html>
<html>
<head>
  <title>gulp-inject</title>
  <!-- inject:css -->
  <!-- endinject -->
</head>
<body>

  <!-- inject:js -->
  <!-- endinject -->
</body>
</html>