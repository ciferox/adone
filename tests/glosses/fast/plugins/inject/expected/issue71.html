<!DOCTYPE html>
<html>
<head>
  <title>gulp-inject</title>
  <!-- inject:css -->
  <link rel="stylesheet" href="/fixtures/styles.css?0123456789abcdef">
  <!-- endinject -->
</head>
<body>

  <!-- inject:js -->
  <script src="/fixtures/lib.js?abcdef0123456789"></script>
  <!-- endinject -->
</body>
</html>